integration-cli/docker_utils_test.go

```diff
@@ -440,7 +440,7 @@ func pollCheck(t *testing.T, f checkF, compare func(x interface{}) assert.BoolOr
 		default:
 			panic(fmt.Errorf("pollCheck: type %T not implemented", r))
 		}
-		return poll.Continue(comment)
+		return poll.Continue(comment) //nolint:govet // ignore non-constant format string in call
```

Comment:
That's a legit lint issue! Fix it; don't just ignore it.
```suggestion
		return poll.Continue("%v", comment)
```

Reply:
Yeah, I was on the fence on these; basically these were designed to either take a literal value, or a format-string with optional arguments. Similar to how (e.g.) `gotest.tools` `assert.Check()` allows a comment with _optional_ formatting.